experiments/nf_evaluation/models.py

```diff
@@ -0,0 +1,30 @@
+from neuralforecast.auto import *
+from neuralforecast.losses.pytorch import HuberLoss, DistributionLoss
+
+
+def get_model(model_name, horizon, num_samples):
+    """Returns the model class given the model name.
+    """
```

Comment:
nit: docstring on one line, i.e., `"""Returns the model class given the model name."""`